### DISCUSSION TITLE
Nebula Graph Index Demystified - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#9] [Comments]
# Nebula Graph Index Demystified - siwei.io

Nebula Index Demystified

https://siwei.io/sketches/nebula-index-demystified/

**Comment by wey-gu** [comment]
update: 后来我又写了一篇文章哈：https://www.siwei.io/nebula-index-explained/ 

另外 https://nebula-graph.com.cn/posts/how-indexing-works-in-nebula-graph/ 这篇第一手知识是最终要的文章。
